src/pages/home/report/ReportFooter.tsx

```diff
@@ -160,6 +177,14 @@ function ReportFooter({
                     {isArchivedRoom && <ArchivedReportFooter report={report} />}
                     {!isArchivedRoom && blockedFromChat && <BlockedReportFooter />}
                     {!isAnonymousUser && !canWriteInReport && isSystemChat && <SystemChatReportFooterMessage />}
+                    {isAdminsOnlyPostingRoom && !isUserPolicyAdmin && !isArchivedRoom && !isAnonymousUser && !blockedFromChat && (
+                        <Banner
```

Comment:
@rushatgabhane while merge I found we have a new feature now called `blocked users` in main branch which will display `<BlockedReportFooter />` in the footer, so I add `!blockedFromChat` to this condition to prevent display both of them at the same time.